dags/main_summary.py

```diff
@@ -203,15 +204,51 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/retention.sh",
     dag=dag)
 
+client_count_daily_base = '''
+normalized_channel,
+country,
+locale,
+app_name,
+app_version,
+e10s_enabled,
+os,
+os_version
+'''.replace('\n', '')
+
+client_count_daily_select = '''
+regexp_extract(subsession_start_date, '(20[0-9]{2}-[0-9]{2}-[0-9]{2})', 1) as activity_date,
+devtools_toolbox_opened_count > 0 as devtools_toolbox_opened,
+case when distribution_id in ('canonical', 'MozillaOnline', 'yandex') then distribution_id else null end as top_distribution_id,
+'''.replace('\n', '') + client_count_daily_base
+
+client_count_daily_groups = '''
+activity_date,
+devtools_toolbox_opened,
+top_distribution_id,
+'''.replace('\n', '') + client_count_daily_base
+
 client_count_daily_view = EMRSparkOperator(
     task_id="client_count_daily_view",
     job_name="Client Count Daily View",
     execution_timeout=timedelta(hours=10),
     owner="relud@mozilla.com",
     email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
     instance_count=2,
-    env = {"date": "{{ ds_nodash }}", "bucket": "{{ task.__class__.private_output_bucket }}"},
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/client_count_daily_view.sh",
+    env = {
+        "arguments": json.dumps({
```

Comment:
How about constructing this configuration struct as utility function like [mozetl_envvar](https://github.com/mozilla/telemetry-airflow/blob/master/dags/utils/mozetl.py). Users won't have to worry about serializing to the correct format, and you can change the underlying implementation as needed.

Reply:
It might also be nice to drop the leading command prefix and add is as part of the utility function, but this is mostly stylistic.

Reply:
fixed